gui/templates/channel.html

```diff
@@ -245,10 +245,10 @@ <h2>Last 5 <a href="/rebalancing" target="_blank">Rebalance Requests</a></h2>
     {% for rebalance in rebalances %}
     <tr>
       <td title="{{ rebalance.requested }}">{{ rebalance.requested|naturaltime }}</td>
-      <td {% if rebalance.status == 0 %}>N/A{% else %}title="{{ rebalance.start }}">{{ rebalance.start|naturaltime }}{% endif %}</td>
-      <td {% if rebalance.status > 1 %}title="{{ rebalance.stop }}">{{ rebalance.stop|naturaltime }}{% else %}>N/A{% endif %}</td>
+      <td {% if rebalance.status == 0 %}>---{% else %}title="{{ rebalance.start }}">{{ rebalance.start|naturaltime }}{% endif %}</td>
+      <td {% if rebalance.status > 1 %}title="{{ rebalance.stop }}">{{ rebalance.stop|naturaltime }}{% else %}>---{% endif %}</td>
       <td>{{ rebalance.duration }} minutes</td>
-      <td {% if rebalance.status == 2 %}>{{ rebalance.stop|timeuntil:rebalance.start }}{% else %}N/A{% endif %}</td>
+      <td {% if rebalance.status == 2 %}>{{ rebalance.stop|timeuntil:rebalance.start }}{% else %}---{% endif %}</td>
```

Comment:
A few spots still where the `>` is missing on one side of the if statement. On this particular line it could be put outside the if statement since there is no hover-over title being used.

Reply:
sorry about that. it seems it was flowing from the previous changes to provide actual time for rebalance. 
https://github.com/cryptosharks131/lndg/pull/108/commits/849fb80bf5238f1edc87fd61f5a87a1100407c54